### PR TITLE
Fix #1222: Update DB scripts for migration to release 1.6.0

### LINKED ISSUE #1222: Update DB scripts for migration to release 1.6.0
Some of the migration scripts are missing: https://github.com/wultra/powerauth-server/blob/develop/docs/PowerAuth-Server-1.6.0.md

### PR DESCRIPTION
**Pull request by zcgandcomp**

### CODE REVIEW

File: docs/sql/postgresql/migration_1.5.3_1.6.0.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+-- Add activation_id column to pa_operation with foreign key constraint
+ALTER TABLE pa_operation ADD activation_id VARCHAR(37);
+
+ALTER TABLE pa_operation ADD CONSTRAINT pa_operation_activation_id_fk FOREIGN KEY (activation_id) REFERENCES pa_activation (activation_id);
+
+-- Changeset powerauth-java-server/1.6.x/20231123-operation-user-nullable.xml::1::Roman Strobl
+-- Make user_id column in table pa_operation nullable
+ALTER TABLE pa_operation ALTER COLUMN  user_id DROP NOT NULL;
+
+-- Changeset powerauth-java-server/1.6.x/20231212-add-tag-1.6.0.xml::1::Lubos Racansky
```
Comment (jnpsk) [inline docs/sql/postgresql/migration_1.5.3_1.6.0.sql:25]: A comment without actual statement? Shouldn't here be some `DATABASECHANGELOG` insert?

File: docs/sql/postgresql/migration_1.5.3_1.6.0.sql [hunk 2]
```diff
@@ -0,0 +1,25 @@
+-- Changeset powerauth-java-server/1.6.x/20231018-add-constraint-operation-template-name.xml::1::Jan Pesek
+-- Add unique constraint to pa_operation_template.template_name
+ALTER TABLE pa_operation_template ADD CONSTRAINT pa_operation_template_template_name_uk UNIQUE (template_name);
+
+-- Changeset powerauth-java-server/1.6.x/20231103-add-activation-name-history.xml::1::Lubos Racansky
+-- Add activation_name column to pa_activation_history
+ALTER TABLE pa_activation_history ADD activation_name VARCHAR(255);
```
Comment (jnpsk) [inline docs/sql/postgresql/migration_1.5.3_1.6.0.sql:7]: This change is not mentioned in migration guide 👀

**Review by jnpsk — COMMENTED** [review]
is it intentional?

File: docs/sql/postgresql/migration_1.5.3_1.6.0.sql (hunk 1 above)
Comment (banterCZ) [reply docs/sql/postgresql/migration_1.5.3_1.6.0.sql:25]: This generated script does not insert anything into `databasechangelog`, even the tag.

File: docs/sql/postgresql/migration_1.5.3_1.6.0.sql (hunk 2 above)
Comment (banterCZ) [reply docs/sql/postgresql/migration_1.5.3_1.6.0.sql:7]: A nice catch, I will add it.

**Review by jnpsk — APPROVED** [review]
Looks ok

File: docs/db/changelog/changesets/powerauth-java-server/1.6.x/20231123-operation-user-nullable.xml [hunk 3]
```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.20.xsd">
```
Comment (zcgandcomp) [inline docs/db/changelog/changesets/powerauth-java-server/1.6.x/20231123-operation-user-nullable.xml:4]: is this necessary?

Comment (banterCZ) [reply docs/db/changelog/changesets/powerauth-java-server/1.6.x/20231123-operation-user-nullable.xml:4]: Already merged. I was missing `columnDataType` in the namespace, but XSD has not been fetched. Anyway, it does not harm.

Comment (zcgandcomp) [reply docs/db/changelog/changesets/powerauth-java-server/1.6.x/20231123-operation-user-nullable.xml:4]: It actually breaks the checksum (I guess with the adding column data type), so yes, this did some harm...

**Pull request merged**